crates/wit-parser/src/ast/resolve.rs

```diff
@@ -560,13 +563,22 @@ impl<'a> Resolver<'a> {
             match field {
                 ast::InterfaceItem::Value(value) => {
                     let docs = self.docs(&value.docs);
+                    let name = value.name();
+                    let span = value.span();
                     match &value.kind {
+                        ValueKind::Func(func) if matches!(value.name, VariableId::Wildcard(_))=> {
+                            let func = self.resolve_function(docs, name, func)?;
+                            let prev = self.interfaces[interface_id]
+                                .wildcard
+                                .replace(func);
+                            assert!(prev.is_none());
```

Comment:
I think this'll need to be a first-class error since I think `*: func()` can show up twice textually?

Reply:
Ahh great catch!